octopoes/octopoes/core/service.py

```diff
@@ -45,7 +45,27 @@
 
 logger = getLogger(__name__)
 settings = Settings()
-BIT_CACHE: dict[int, list[OOI]] = {}
+
+
+class BitCache:
+    def __init__(self):
+        self.bit_cache: dict[int, tuple[list[OOI], datetime]] = {}
+
+    def __getitem__(self, key: int):
```

Comment:
Missing return type

Reply:
Not relevant for a9ff441948996c1e6d66a3464c5d5e2f95088da9